src/types/voice.rs

```diff
@@ -1,12 +1,17 @@
-#[derive(Debug, Deserialize, Clone, PartialEq, Eq, Hash)]
 /// This object represents a voice note.
+///
+/// [The official docs](https://core.telegram.org/bots/api#voice).
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub struct Voice {
-    /// Identifier for this file
+    /// Identifier for this file.
     pub file_id: String,
-    /// Duration of the audio in seconds as defined by sender
+
+    /// Duration of the audio in seconds as defined by sender.
     pub duration: u32,
-    /// Optional. MIME type of the file as defined by sender
+
+    /// Optional. MIME type of the file as defined by sender.
     pub mime_type: Option<String>,
```

Comment:
Could this be an enum?...

Reply:
https://docs.rs/mime/0.3.14/mime/ ?

Reply:
Maybe. Also, I've just seen that you missed `Optional. ` here

Reply:
Where?

Reply:
Here
![image](https://user-images.githubusercontent.com/38225716/70647890-00d85280-1c5b-11ea-9f29-2c09f8bad5a0.png)

Reply:
Ah, I'll fix it.

Reply:
An in lots of other places...